verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -208,7 +211,7 @@ mod tests {
             "deployed_bytecode": "0x6001",
             "creation_bytecode": "0x6001",
             "compiler_version": "v0.8.2+commit.661d1103",
-            "input":{"language":"Solidity","sources":{"./src/contracts/Foo.sol":{"content":"pragma solidity ^0.8.2;\n\ncontract Foo {\n    function bar() external pure returns (uint256) {\n        return 42;\n    }\n}\n"}},"settings":{"metadata":{"useLiteralContent":true},"optimizer":{"enabled":true,"runs":200},"outputSelection":{"*":{"*":["abi","evm.bytecode","evm.deployedBytecode","evm.methodIdentifiers"],"":["id","ast"]}}}}
```

Comment:
Why? I suggest to better add into test that the valie StandardJson could be converted into CompilerInput